upgrade/management_nodes_and_csm_services/Stage_0_Prerequisites.md

```diff
@@ -96,7 +96,7 @@ backup of Workload Manager configuration data and files is created. Once complet
 1. Run the script.
 
    ```bash
-   ncn-m001# /usr/share/doc/csm/upgrade/1.2/scripts/upgrade/prepare-assets.sh --csm-version ${CSM_RELEASE} --tarball-file "${CSM_TAR_PATH}"
+   ncn-m001# /usr/share/doc/csm/upgrade/management_nodes_and_csm_services/scripts/upgrade/prepare-assets.sh --csm-version ${CSM_RELEASE} --tarball-file "${CSM_TAR_PATH}"
```

Comment:
Can you please fix-up these prompts to match the `main` branch convention? https://github.com/Cray-HPE/docs-csm/blob/MTL-1695/introduction/documentation_conventions.md#step-examples